features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Listing/ApiThumb.jsx

```diff
@@ -245,19 +171,28 @@ class ApiThumb extends React.Component {
                         </div>
                     </div>
                     <div className={classes.thumbInfo}>
-                        <Typography variant='subheading' gutterBottom align='left'>
-                            <StarRatingBar rating={rating} starColor={starColor} />
-                        </Typography>
+                        <div className={classes.thumbLeft}>
+                            <Typography variant='subheading' gutterBottom align='left'>
+                                <StarRatingBar rating={rating} starColor={starColor} />
+                            </Typography>
+                        </div>
+                        <div className={classes.thumbRight}>
+                            <Typography variant='subheading' gutterBottom align='right'>
+                                {api.type === 'GRAPHQL' && (
+                                    <Chip label={api.type} color='primary' />
+                                )}
+                            </Typography>
+                        </div>
                     </div>
                 </div>
-            </div>
+            </Grid>
         );
     }
 }
 
 ApiThumb.propTypes = {
-    classes: PropTypes.shape({}).isRequired,
-    theme: PropTypes.shape({}).isRequired,
+    classes: PropTypes.object.isRequired,
+    theme: PropTypes.object.isRequired,
```

Comment:
doing this would cause eslint error. Previous shape({}) is correct way. Additonally we can check for the properties within the shape too

Reply:
Fixed